src/actions/effect/leveled/MakeTransparent.ts

```diff
@@ -40,7 +41,7 @@ class MakeTransparentEffectAction extends LeveledEffectAction {
 
     // We are using this() to allow inheriting classes to use super.fromJson.apply(this, [actionModel])
     // This allows the inheriting classes to determine the class to be created
-    const result = new this(actionType, tolerance);
+    const result = new this(ACTION_TYPE_TO_EFFECT_MODE_MAP[actionType], tolerance);
```

Comment:
@strausr , how come this change does not affect other tests? like `unsharpMask`, `advancedRedEye`, etc?

Reply:
if it does, maybe it's reasonable to add other tests as well?

Reply:
The reason is that those variables were already handled in our internal mapping 
(See src/internal/internalConstants.ts)

This PR added `makeTransparent: 'make_transparent'`

Reply:
OK